AstronomyUITests/AstronomyUITests.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  AstronomyUITests.swift
+//  AstronomyUITests
+//
+//  Created by Rob Vance on 9/24/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import XCTest
+@testable import Astronomy
+
+class AstronomyUITests: XCTestCase {
+    
+    var app: XCUIApplication {
+        return XCUIApplication()
+    }
+    
```

Comment:
Did you delete your `setUpWithError()` method? That would keep you from having to say app.launch() each time. You should have had these two methods:
``` swift
override func setUpWithError() throws {
    app = XCUIApplication()
    app.launchArguments = ["UITesting"]
    app.launch()
    
    continueAfterFailure = false
}

override func tearDownWithError() throws {
    app = nil
}
```